### PR TITLE
Delete metric clean up

### PR DESCRIPTION
**Pull request by cdsgibson**
- Modified the "create" methods in MetricDelController and MetricDelInputBoundary to "deleteMetric"
- replaced instances of "create" with "deleteMetric"
- fixed the Java Doc for deleteMetric  in MetricDelController
- added setUp to MetricDeleterTest

### CODE REVIEW

File: src/main/java/presenters/MetricDelOutputBoundary.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Interface for the presenter that handles the deletion of a metric.
  */
 
-public interface MetricDelOutputBoundary {
+public interface    MetricDelOutputBoundary {
```
Comment (craftedswift) [inline src/main/java/presenters/MetricDelOutputBoundary.java:9]: is the extra indentation a typo?

**Review by craftedswift — CHANGES_REQUESTED** [review]
fix the typo in the metricdecl output boundary

File: src/main/java/presenters/MetricDelOutputBoundary.java (hunk 1 above)
Comment (cdsgibson) [reply src/main/java/presenters/MetricDelOutputBoundary.java:9]: Yes, it is, my bad. I will fix it.

**Review by craftedswift — APPROVED** [review]
looks good!

**Review by Bensonwu02 — APPROVED** [review]
looks good. thanks for making those changes.

**Review by Wizard603 — APPROVED** [review]
Glad to see the metricDeleter is no longer called metricCreator

**Pull request merged**